packages/control-property-editor/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "@sap-ux/control-property-editor",
+  "displayName": "SAP Fiori Tools - Control Property Editor",
+  "description": "SAP Fiori Tools - Control Property Editor",
+  "license": "SEE LICENSE IN LICENSE",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SAP/open-ux-tools.git",
+    "directory": "packages/control-property-editor"
+},
+  "private": false,
+  "scripts": {
+    "pre-commit": "lint-staged --quiet",
+    "clean:dist": "rimraf ./dist ./generators *.tsbuildinfo",
+    "clean": "rimraf ./out ./reports ./.webpack_cache",
+    "clean:all": "npm-run-all clean",
+    "build": "npm-run-all -l -s clean -p build:compile build:webapp",
+    "build:compile": "tsc  --build ./tsconfig.build.json --pretty --dry",
+    "build:webapp": "node esbuild.js",
+    "watch": "npm-run-all -p watch:*",
+    "watch:webapp": "node esbuild.js --watch --minify=false",
+    "start": "node development/server",
+    "test": "jest --maxWorkers=1 --ci --forceExit --detectOpenHandles --config='jest.config.js'",
+    "test:integration": "jest --maxWorkers=1 --ci --config='jest.ui5.config.js'",
+    "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand  --config='jest.config.js' --colors",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:fix": "eslint . --ext .ts,.tsx --fix",
+    "format:fix": "prettier --write --ignore-path ../../../.prettierignore",
+    "format:fix:all": "prettier --write '**/*.{css,scss,html,js,json,ts,tsx,yaml,yml}' '!**/{out,dist,typings,node_modules}/**' '!**/*.{svg,png,xml}' --ignore-path ../../../.prettierignore",
+    "madge": "madge --warning --circular --extensions ts ./",
+    "setup": "./setup.sh"
+  },
+  "devDependencies": {
+    "@fluentui/react": "8.100.0",
+    "@fluentui/react-hooks": "8.6.14",
+    "@reduxjs/toolkit": "1.6.1",
+    "@sap-ux/ui-components": "1.9.0",
+    "@sapui5/ts-types": "1.90.6",
```

Comment:
You don't need `"@sapui5/ts-types"` your are getting these through the `types` reference in the `tsconfig`.

Reply:
removed.